test/sqlancer/dbms/TestPostgresNoREC.java

```diff
@@ -0,0 +1,23 @@
+package sqlancer.dbms;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+import org.junit.jupiter.api.Test;
+
+import sqlancer.Main;
+
+public class TestPostgresNoREC {
+
+    String postgresAvailable = System.getenv("POSTGRES_AVAILABLE");
```

Comment:
Do you think it might be worthwhile to create a PostgreSQL test base class in which we can move the `postgresIsAvailable` logic?

Reply:
maybe instead of just for postgres, we can maybe add a utility function that can be used by all databses into TestConfig?

Reply:
i added a commit which shows the change. If this is good then I can also make the change for the other databases, either in this pr or a separate one.

Reply:
Looks good. Do you think we should make "POSTGRES_AVAILABLE" a string constant in `TestConfig`? This would help to avoid any issues if, at some point in time, we rename the constant. Either is fine to me - as you prefer.

Reply:
I think that's a good idea. I've made the change.

Also, I think I will make the changes to other databases in another PR